js/src/dapps/registry/Names/names.js

```diff
@@ -78,35 +79,21 @@ const renderQueue = (queue) => {
 class Names extends Component {
 
   static propTypes = {
+    error: nullableProptype(PropTypes.object.isRequired),
```

Comment:
It would make sense to write it that IMO, however it doesn't work. It's done like this everywhere else in the app.